### PR TITLE
Hosb 12

### PR DESCRIPTION
**Pull request by MaxFuhrich**

### CODE REVIEW

File: osb-backup-manager/src/main/java/de/evoila/cf/backup/service/manager/BackupServiceManager.java [hunk 1]
```diff
@@ -43,6 +43,7 @@ public BackupServiceManager(AbstractJobRepository abstractJobRepository,
         this.abstractJobRepository = abstractJobRepository;
         this.credentialService = credentialService;
         this.backupCleanupManager = backupCleanupManager;
+        log.debug("Debug activated");
```
Comment (miriSch) [inline osb-backup-manager/src/main/java/de/evoila/cf/backup/service/manager/BackupServiceManager.java]: should this be removed?

File: osb-backup-manager/src/main/java/de/evoila/cf/backup/service/manager/BackupServiceManager.java [hunk 2]
```diff
@@ -133,56 +135,63 @@ private void executeBackup(BackupExecutorService backupExecutorService, Endpoint
...
                     if (result != null) {
                         if (result.getStatus().equals(JobStatus.SUCCEEDED)) {
                             backupJob.getFiles().put(item, result.getFilename());
                         }
 
                         updateWithAgentResponse(backupJob, item, result);
                     }
-
                     checkFuture.cancel(true);
-                    log.info("Finished execution of Backup Job");
```
Comment (miriSch) [inline osb-backup-manager/src/main/java/de/evoila/cf/backup/service/manager/BackupServiceManager.java:169]: was the deletion of this log line on purpose?

File: osb-backup-manager/src/main/resources/logback-spring.xml [hunk 3]
```diff
@@ -8,6 +8,7 @@
 		<appender-ref ref="CONSOLE" />
 	</root>
 
+	<logger name="de.evoila.cf" level="DEBUG" />
```
Comment (miriSch) [inline osb-backup-manager/src/main/resources/logback-spring.xml]: should this be removed?

File: osb-backup-manager/src/main/java/de/evoila/cf/backup/service/manager/BackupServiceManager.java (hunk 1 above)
Comment (MaxFuhrich) [reply osb-backup-manager/src/main/java/de/evoila/cf/backup/service/manager/BackupServiceManager.java]: The log was added by me when we started looking for the problem. Thats why i removed it now. We can keep it if you find it useful

File: osb-backup-manager/src/main/java/de/evoila/cf/backup/service/manager/BackupServiceManager.java (hunk 2 above)
Comment (MaxFuhrich) [reply osb-backup-manager/src/main/java/de/evoila/cf/backup/service/manager/BackupServiceManager.java:169]: Yes, for the same reason as above

Comment (miriSch) [reply osb-backup-manager/src/main/java/de/evoila/cf/backup/service/manager/BackupServiceManager.java:169]: this log line has been added before version 0.0.8

File: osb-backup-manager/src/main/java/de/evoila/cf/backup/service/manager/BackupServiceManager.java (hunk 1 above)
Comment (miriSch) [reply osb-backup-manager/src/main/java/de/evoila/cf/backup/service/manager/BackupServiceManager.java]: You did not remove this log line. That's why I suggested to remove it.

File: osb-backup-manager/src/main/java/de/evoila/cf/backup/service/manager/BackupServiceManager.java (hunk 2 above)
Comment (MaxFuhrich) [reply osb-backup-manager/src/main/java/de/evoila/cf/backup/service/manager/BackupServiceManager.java:169]: Ok then we should keep it

**Review by miriSch — APPROVED** [review]

**Pull request merged**